modules/heat_conduction/doc/content/source/actions/ThermalContactAction.md

```diff
@@ -0,0 +1,14 @@
+# ThermalContactAction
+
+!alert construction
+Documentation for the heat conduction module needs some work...
```

Comment:
Yeah it does...

For actions, we end up with two pages: one for the syntax block and one for the action itself. We are putting all of the documentation in the syntax pages, so just replace all of what you have currently in this .md file with the text below that points the user to the (not yet populated) syntax page:

```
# Thermal Contact Action

## Description

The `ThermalContactAction` action sets up the set of models used
to enforce thermal contact across two surfaces. See the description,
example use, and parameters on the
[ThermalContact](/ThermalContact/index.md) system documentation page.